# HG changeset patch
# Parent d8c77ce3f69f380dbf798b3ccae814cff33ec781
# User Frédéric Wang <fred.wang@free.fr>
Use MathJax fonts on token elements. Bug ??????

diff --git a/layout/mathml/mathml.css b/layout/mathml/mathml.css
--- a/layout/mathml/mathml.css
+++ b/layout/mathml/mathml.css
@@ -49,27 +49,30 @@
 /**************************************************************************/
 
 math {
   direction: ltr;
   unicode-bidi: embed;
   display: inline;
   font-size: inherit;
   font-style: normal;
-  font-family: STIXGeneral, DejaVu Serif, DejaVu Sans, Cambria, Cambria Math, Times, Lucida Sans Unicode, OpenSymbol, Standard Symbols L, serif;
+  font-family: MathJax_Main, STIXGeneral, DejaVu Serif, DejaVu Sans, Cambria, Cambria Math, Times, Lucida Sans Unicode, OpenSymbol, Standard Symbols L, serif;
   text-rendering: optimizeLegibility;
   -moz-float-edge: margin-box;
 }
 math[mode="display"], math[display="block"] {
   display: block;
   text-align: -moz-center;
 }
 math[display="inline"] {
   display: inline;
 }
+mi {
+  font-family: MathJax_Math, STIXGeneral, DejaVu Serif, DejaVu Sans, Cambria, Cambria Math, Times, Lucida Sans Unicode, OpenSymbol, Standard Symbols L, serif;
+}
 
 /**************************************************************************/
 /* Style switching during frame construction depending on the context of <mi>:
    These rules are not used when mathvariant or fontstyle is specified
    explicitly. 
 /**************************************************************************/
 
 /* If the textual content of an <mi> consists of a single character
@@ -170,33 +173,33 @@ math[display="inline"] {
 [mathvariant="italic"] { 
   font-style: italic;
 }
 [mathvariant="bold-italic"] {
   font-weight: bold;
   font-style: italic;
 }
 [mathvariant="sans-serif"] {
-  font-family: sans-serif;
+  font-family: MathJax_SansSerif, sans-serif !important;
 }
 [mathvariant="bold-sans-serif"] {
   font-weight: bold;
-  font-family: sans-serif;
+  font-family: MathJax_SansSerif, sans-serif !important;
 }
 [mathvariant="sans-serif-italic"] {
-  font-family: sans-serif;
+  font-family: MathJax_SansSerif, sans-serif !important;
   font-style: italic;
 }
 [mathvariant="sans-serif-bold-italic"] {
-  font-family: sans-serif;
+  font-family: MathJax_SansSerif, sans-serif !important;
   font-weight: bold;
   font-style: italic;
 }
 [mathvariant="monospace"] {
-  font-family: monospace;
+  font-family: MathJax_Typewriter, monospace !important;
 }
 
 /**************************************************************************/
 /* tags with generated content                                            */
 /**************************************************************************/
 
 ms {
   display: inline;
